frontend/src/fileSender.js

```diff
@@ -64,6 +65,7 @@ class FileSender extends EventEmitter {
           window.crypto.subtle.exportKey('jwk', secretKey)
         ]);
       })
+      .catch(err => { Raven.captureException(err) })
```

Comment:
This _may_ have some side effects, IIRC.
I think `.catch()` will handle/swallow any errors and then proceed onto the next `.then()`, which may get some null/undefined args.
So some things may fail, but not 100% sure what will happen and if it may rethrow errors and trigger the `.catch()` on L105 below and resend some errors to Raven.